spec/components/password_confirmation_component_spec.rb

```diff
@@ -20,4 +20,24 @@
       type: :checkbox,
     )
   end
+
+  context 'with labels passed in' do
+    subject(:rendered) do
+      render_inline PasswordConfirmationComponent.new(
+        form:,
+        password_label: 'edited password label',
+        confirmation_label: 'edited password confirmation label',
```

Comment:
Should we use the `let` variables here, to avoid duplicating?

```suggestion
        password_label:,
        confirmation_label:,
```